_validate/validate.py

```diff
@@ -223,22 +230,39 @@ def checkManifestVersionMatchesVersionName(
 		)
 
 
+def checkLastTestedVersionExist(submission: JsonObjT, verFilename: str) -> ValidationErrorGenerator:
+	lastTestedVersion: JsonObjT = submission['lastTestedVersion']
+	formattedLastTestedVersion: str = _formatVersionString(lastTestedVersion.values())
+	if formattedLastTestedVersion not in getExistingVersions(verFilename):
+		yield f"Last tested version error: {formattedLastTestedVersion} doesn't exist"
+
+
+def checkMinRequiredVersionExist(submission: JsonObjT, verFilename: str) -> ValidationErrorGenerator:
```

Comment:
I think this check should be the same as `checkLastTestedVersionExist`, it should check if the version exists and nothing else.
I'm not sure I understand the current logic.